src/mono/mono/profiler/CMakeLists.txt

```
@@ -28,6 +28,13 @@ if(NOT DISABLE_LIBS)
     endif()
   endif()
 
+  if(HOST_ANDROID)
+    add_library(mono-profiler-aot SHARED aot.c helper.c)
+    target_compile_definitions(mono-profiler-aot PRIVATE -DMONO_DLL_EXPORT)
+    target_link_libraries(mono-profiler-aot monosgen-shared eglib_objects log)
```

@directhex 

```suggestion
    target_link_libraries(mono-profiler-aot monosgen-shared eglib_objects log monoapi)
```

That should fix the broken builds.  (Fallout from #64569)

---

(And possibly the same thing for `mono-profiler-log`, above